### ISSUE TITLE
cannot launch solara

### ISSUE THREAD

**Issue by hun-kun** [quivings/Solara#50]
i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked

**Comment by Skeleton19** [comment]
> i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked

What error is appearing?

**Comment by bfdifan1** [comment]
> > i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked
> 
> What error is appearing?

for me nothing

**Comment by bfdifan1** [comment]
> > i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked
> 
> What error is appearing?

it just opens and closes but doesnt lauch the exectuter

**Comment by Skeleton19** [comment]
> > > i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked
> > 
> > 
> > What error is appearing?
> 
> it just opens and closes but doesnt lauch the exectuter

Do you have node.js?

**Comment by bfdifan1** [comment]
> > > > i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked
> > > 
> > > 
> > > What error is appearing?
> > 
> > 
> > it just opens and closes but doesnt lauch the exectuter
> 
> Do you have node.js?

i dont think so but before that exectuter changed my wallpaper to a dick picture and a note popup and said all my files are enecryted and i shittedr my pants

**Comment by bfdifan1** [comment]
![image](https://github.com/quivings/Solara/assets/130241472/50ff8c3d-c700-4bcc-8a2e-3b371b3289e6)

![image](https://github.com/quivings/Solara/assets/130241472/835afd0a-ef4b-450f-878c-0ca0ddb9eb94)

**Comment by bfdifan1** [comment]
i know why

**Comment by bfdifan1** [comment]
![image](https://github.com/quivings/Solara/assets/130241472/cba25223-094e-42f9-bac9-2190cbdaa8b5)

**Comment by bfdifan1** [comment]
> it

hello?

**Comment by bfdifan1** [comment]
hewlp me pls

**Comment by bfdifan1** [comment]
im scared!

**Comment by bfdifan1** [comment]
PLS THE VIRUS IS CHAGING EVERYTHING HELP

**Comment by bfdifan1** [comment]
https://imgs.search.brave.com/Rbm4lfwwlALK2UptNfOXlAKXfxzyLM6jLLwCQutFbFs/rs:fit:860:0:0:0/g:ce/aHR0cHM6Ly9pMS5z/bmRjZG4uY29tL2Fy/dHdvcmtzLTAwMDQ3/MTcxMzY4Mi0yZzUx/dmUtdDUwMHg1MDAu/anBn

**Comment by Skeleton19** [comment]
> > > > > i have done everything my friend, some kid on yt and the guide for fixing the thing is but none has worked
> > > > 
> > > > 
> > > > What error is appearing?
> > > 
> > > 
> > > it just opens and closes but doesnt lauch the exectuter
> > 
> > 
> > Do you have node.js?
> 
> i dont think so but before that exectuter changed my wallpaper to a dick picture and a note popup and said all my files are enecryted and i shittedr my pants

you need node.js, this program needs for opening ui.
